### ISSUE TITLE
Add comparison to Jupyter in README

### ISSUE THREAD

**Issue by vassudanagunta** [gpoore/codebraid#2]
Many people will wonder why codebraid instead of [Jupyter](https://jupyter.org). Is it all about Pandoc?

It makes sense to address that in the README.

**Comment by sterlinm** [comment]
I'm also curious about codebraid versus R markdown. Was that an inspiration at all?

**Comment by piccolbo** [comment]
Two projects to compare with for me would be knitr and pweave. knitr is supposed to be multi-language, but I think the traction is mostly in R circles. pweave is python-only and kind of unofficially maintenance-only right now, but it's what I use.

**Comment by gpoore** [comment]
Sorry for the delay, everyone. I now have enough features working to add a decent comparison that shows some of Codebraid's unique capabilities.

@vassudanagunta Adding a comparison is a good idea. I've added a table and a short paragraph in the README that address Jupyter. If you have questions that this doesn't address, I'd be happy to discuss that and possibly expand the comparison.

@sterlinm I've added a comparison that includes knitr/R markdown. I wouldn't exactly call it an inspiration, but it has influenced my work somewhat since it's such a significant executable-code-in-markdown program. I'm coming to this after working with LaTeX for years with [PythonTeX](https://github.com/gpoore/pythontex), and then wanting similar markdown capabilities.

@piccolbo I've added a comparison that includes knitr and pweave. Codebraid already has some features that knitr and pweave lack, such as multiple independent sessions per language. However, it doesn't have automatic figure saving and inclusion, and I still need to make a decision about the extent to which I want to support those sorts of things.
